lib/tasks/local_service.rake

```diff
@@ -0,0 +1,6 @@
+namespace :local_service do
+  desc "Changes an existing local_service's LGSL code"
+  task :change_lgsl_code, %w[from_lgsl to_lgsl] => :environment do |_, args|
+    ChangeLgslCodeService.call(args.from_lgsl, args.to_lgsl)
+  end
```

Comment:
```suggestion
  desc "Update the temporary LGSL code of 100001"
  task change_lgsl_code_10001: :environment do
    draft = LocalTransactionEdition.find_by!(lgsl_code: 100001, state: :draft)
    LocalService.find_by!(lgsl_code: 100001).update!(lgsl_code: 1827)
    draft.update!(lgsl_code: 1827)
  end
```

I think we can rather drastically simplify this and tailor this just for this specific scenario, this should probably remove the need to be so defensive in the code and make this a quicker thing to get out, run and then removed.